user/languages/r.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Building an R Project
+layout: en
+permalink: /user/languages/r/
+---
+
+### What This Guide Covers
+
+This guide covers build environment and configuration topics specific to R
+projects. Please make sure to read our
+[Getting Started](/user/getting-started/) and
+[general build configuration](/user/build-configuration/) guides first.
+
+### Community-Supported Warning
+
+Travis CI support for R is contributed by the community and may be removed or
+altered at any time. If you run into any problems, please report them in the
+[Travis CI issue tracker](https://github.com/travis-ci/travis-ci/issues/new?labels=community:r)
+and cc [@craigcitro](https://github.com/craigcitro),
+[@eddelbuettel](https://github.com/eddelbuettel), and
+[@hadley](https://github.com/hadley).
+
+## Basic configuration
+
+If your R repository doesn't need any dependencies beyond those specified in
```

Comment:
Added an introductory sentence, but I don't know if I should say more. (I'm tempted to also link to your book ...)